perl/INSTALL.md

```diff
@@ -0,0 +1,176 @@
+# Bystro Annotator Package Installation
+
+## Installing Bystro Annotator using Docker
+
+To build a Docker image using the `Dockerfile`, run the following:
+
+```bash
+cd ../ && docker build -t bystro-annotator -f Dockerfile.perl .
+```
+
```

Comment:
Similar to the section below, I would like to have instructions for trying it after the docker install is done, in case I don't know how docker works.

Now you're ready to try Bystro:
```bash
# First let's run our test suite
cd ~/bystro/perl
prove -r ./t -j$(nproc)

# Then let's try running bystro-annotate.pl
bystro-annotate.pl --help
```

Reply:
done!